src/utils/string/index.ts

```diff
@@ -161,3 +161,17 @@ export const sortFunctionDate = (a, b, key) => {
  * @return {string} A regex that strictly matches on the input.
  */
 export const applyStrictRegex = (str: string) => `^${str}$`;
+
+/**
+ * @param {string} word - A singular string to pluralize
+ * @param {number} count - A count to base pluralization
+ */
+export const pluralize = (word: string, count: number) => {
```

Comment:
Nice 😁 I think the English language is too complicated for this to handle every case we'll run into, but nice to have some standard uses covered here!